mcs_benchmark_data/cli/commands/create_submission_pipeline_command.py

```diff
@@ -58,76 +62,50 @@ def __call__(self, args):
         else:
             data_dir = "DATA_DIR_PATH"
 
-        is_first_submission = self.make_submission_directories(
-            root_path=root_path,
-            benchmark_name=benchmark_name,
-            submission_name=submission_name,
-        )
-
-        self.create_files_from_template(
-            root_path=root_path,
+        is_first_submission = self._make_submission_directories(
+            root_path=ROOT_DIR_PATH,
             benchmark_name=benchmark_name,
             submission_name=submission_name,
-            data_dir=data_dir,
-            is_first_submission=is_first_submission,
         )
 
-    def create_files_from_template(
-        self,
-        *,
-        root_path: Path,
-        benchmark_name: str,
-        submission_name: str,
-        data_dir: str,
-        is_first_submission: bool,
-    ):
-
-        format_args = {
-            "benchmark_name": benchmark_name,
-            "BenchmarkName": sc.capitalcase(benchmark_name),
-            "submission_name": submission_name,
-            "SubmissionName": sc.capitalcase(submission_name),
-            "data_dir": data_dir,
-        }
-
         self._create_files_from_template(
-            root_path=root_path,
+            root_path=ROOT_DIR_PATH,
             benchmark_name=benchmark_name,
             submission_name=submission_name,
             data_dir=data_dir,
-            format_args=format_args,
             is_first_submission=is_first_submission,
         )
 
-    def make_submission_directories(
+    def _make_submission_directories(
         self, root_path: Path, benchmark_name: str, submission_name: str

```

Comment:
As with `_make_benchmark_directories`, you'll have both of these parameters attached to `self`.